lib/traject/config/folio_config.rb

```diff
@@ -165,9 +165,21 @@ def holdings(record, context)
 end
 
 to_field 'folio_json_struct' do |record, accumulator|
-  accumulator << JSON.generate(record.record)
+  accumulator << JSON.generate(record.record.except('parsedRecord'))
 end
 
 to_field 'holdings_json_struct' do |record, accumulator|
   accumulator << JSON.generate(record.items_and_holdings) if record.items_and_holdings
 end
+
+
+## Postprocessing
+# https://consul.stanford.edu/display/SYSTEMS/Data+Feed+to+SearchWorks
+
+# Remove "junk tags"
+each_record do |record, context|
+end
+
+# Add date accessed (from client) to marc 005
+each_record do |record, context|
+end
```

Comment:
I'm guessing leaving this here is intended as a placeholder for future work?

Reply:
yep, and no longer relevant. removed it, thx